### PR TITLE
Check point dome9 integration

### PR DESCRIPTION
**Pull request by TalGumi**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
A few sentences describing the overall goals of the pull request's commits.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [x ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [x] Yes
       - Further details:
   - [ ] No

## Must have
- [x ] Tests
- [x ] Documentation

### CODE REVIEW

File: Packs/CheckPointDome9/Integrations/CheckPointDome9/CheckPointDome9.py
Comment (JudahSchwartz) [inline Packs/CheckPointDome9/Integrations/CheckPointDome9/CheckPointDome9.py:782]:
```suggestion
        pagination_message = f'Showing page {page}. \n Current page size: {page_size}'
```

File: Packs/CheckPointDome9/Integrations/CheckPointDome9/CheckPointDome9.py
Comment (JudahSchwartz) [inline Packs/CheckPointDome9/Integrations/CheckPointDome9/CheckPointDome9.py:722]:
```suggestion
    items = []
    if ip_list:
        while len(ip_list) > len(comment_list):
            comment_list.append('')
```

Comment (JudahSchwartz) [reply Packs/CheckPointDome9/Integrations/CheckPointDome9/CheckPointDome9.py:722]: join the two `if ip_list`

File: Packs/CheckPointDome9/Integrations/CheckPointDome9/CheckPointDome9.py
Comment (JudahSchwartz) [inline Packs/CheckPointDome9/Integrations/CheckPointDome9/CheckPointDome9.py:1539]: what are these commented lines? Should they be deleted?

File: Packs/CheckPointDome9/Integrations/CheckPointDome9/CheckPointDome9.py
Comment (JudahSchwartz) [inline Packs/CheckPointDome9/Integrations/CheckPointDome9/CheckPointDome9.py:2054]: Please switch to inline dict

**Review by JudahSchwartz — COMMENTED** [review]
Code looks great.
Lets wait for doc review
Please notice the failing tests/validations

File: Packs/CheckPointDome9/Integrations/CheckPointDome9/CheckPointDome9.py
Comment (TalGumi) [reply Packs/CheckPointDome9/Integrations/CheckPointDome9/CheckPointDome9.py:1062]: Yes :)

File: Packs/CheckPointDome9/Integrations/CheckPointDome9/CheckPointDome9.py
Comment (TalGumi) [reply Packs/CheckPointDome9/Integrations/CheckPointDome9/CheckPointDome9.py:720]: OK :)

File: Packs/CheckPointDome9/Integrations/CheckPointDome9/CheckPointDome9.py
Comment (TalGumi) [reply Packs/CheckPointDome9/Integrations/CheckPointDome9/CheckPointDome9.py:1537]: I deleted them:)

**Review by JudahSchwartz — APPROVED** [review]
Great! Just waiting on docs now.

**Comment by JudahSchwartz** [comment]
@TalGumi can you please give @ShirleyDenkberg write access to your content repo?

**Comment by ShirleyDenkberg** [comment]
@TalGumi I made several changes in the yml file.  Please regenerate the Readme.
@JudahSchwartz Doc review completed.

**Pull request merged**
